data/json/items/melee.json

```diff
@@ -2050,6 +2050,23 @@
         "to_hit": 2
   },{
         "type":"GENERIC",
+        "id" : "naginata",
+        "name" : "naginata",
+        "name_plural": "naginata",
+        "description" : "A sturdy polearm with a curved blade, made in the same manner as the katana and other Japanese blades.  Occasionally used by samurai in early periods, or by their wives in defense of the household.",
+        "weight" : 1700,
+        "to_hit" : 2,
+        "color" : "light_gray",
+        "symbol" : "/",
+        "material" : ["steel", "wood"],
+        "techniques" : ["RAPID", "IMPALE", "WBLOCK_2"],
```

Comment:
I don't think RAPID really fits a naginata. It is no more rapid than a spear and spears no longer have it.
IMPALE doesn't fit because you slice with a naginata, impaling is for stabbing weapons where you pierce the current target and the one behind it.

It could get a small (2 to 5 points, per 100 moves) damage bonus to compensate for losing those.